cogs/spawning.py

```diff
@@ -448,20 +448,26 @@ async def shinyhunt(self, ctx, *, species: str = None):
 
         member = await self.bot.mongo.fetch_member_info(ctx.author)
 
+        # variable for current shiny hunt
+        current_hunting = self.bot.data.species_by_number(member.shiny_hunt).species_by_name
```

Comment:
This doesn't look right

Reply:
oh frick my bad, i meant to put member.shiny_hunt

Reply:
I think i fixed it, could you take a look? i really appreciate the corrections :)